### PR TITLE
feat: calendar colors

### LINKED ISSUE #58: Colors per event on the calendar

### PR DESCRIPTION
**Pull request by LeoDog896**
Closes #58

### CODE REVIEW

**Review by CoasterFan5 — CHANGES_REQUESTED** [review]
Positioning of the color picker is a little weird (tested on chromium based). Also color picker element looks out of place.

**Comment by CoasterFan5** [comment]
@LeoDog896 lmk what you think

File: src/lib/components/ColorPicker.svelte [hunk 1]
```diff
@@ -0,0 +1,182 @@
+<script lang="ts">
+	import { cubicInOut } from 'svelte/easing';
+	import { fly } from 'svelte/transition';
+
+	import PaletteIcon from '~icons/bx/palette';
+	import { clickOutside } from '$lib/actions/clickOutside';
+
+	export let name: string;
+	export let color: string;
+	export let label = 'Customize color';
```
Comment (LeoDog896) [inline src/lib/components/ColorPicker.svelte:10]: title case?

**Review by LeoDog896 — COMMENTED** [review]
looks good! one review. no clue if its correct.

**Pull request merged**